common-api/src/main/java/org/pmiops/workbench/cdr/dao/CriteriaDao.java

```diff
@@ -0,0 +1,166 @@
+package org.pmiops.workbench.cdr.dao;
```

Comment:
Is there a way you can move this file instead of delete it so git can track it and I can see the differences from the old one? like:
`git mv api/src/main/java/org/pmiops/workbench/cdr/dao/CriteriaDao.java  common-api/src/main/java/org/pmiops/workbench/cdr/dao/CriteriaDao.java`

You may need to save  your new file if it is different. Roll back the change and try that commit that. Than paste your new file and commit that. Something like that.

Reply:
Tried that.